scenes/game_elements/props/decoration/water_rock/components/water_rock_fabric.tres

```diff
@@ -0,0 +1,94 @@
+[gd_resource type="SpriteFrames" load_steps=14 format=3 uid="uid://d064khv6df2j4"]
+
+[ext_resource type="Texture2D" uid="uid://bvgio1b8la3bg" path="res://assets/first_party/rocks/RockWater_Idle.png" id="1_7p1i2"]
+[ext_resource type="Texture2D" uid="uid://bbclkccipxec2" path="res://assets/first_party/rocks/RockWater_Struck.png" id="2_h360w"]
+
+[sub_resource type="AtlasTexture" id="AtlasTexture_egmya"]
+atlas = ExtResource("1_7p1i2")
+region = Rect2(0, 0, 128, 128)
+
+[sub_resource type="AtlasTexture" id="AtlasTexture_0mrsi"]
+atlas = ExtResource("1_7p1i2")
+region = Rect2(128, 0, 128, 128)
+
+[sub_resource type="AtlasTexture" id="AtlasTexture_mk0qy"]
+atlas = ExtResource("1_7p1i2")
+region = Rect2(256, 0, 128, 128)
+
+[sub_resource type="AtlasTexture" id="AtlasTexture_jnqno"]
+atlas = ExtResource("1_7p1i2")
+region = Rect2(384, 0, 128, 128)
+
+[sub_resource type="AtlasTexture" id="AtlasTexture_yh4pc"]
+atlas = ExtResource("1_7p1i2")
+region = Rect2(512, 0, 128, 128)
+
+[sub_resource type="AtlasTexture" id="AtlasTexture_kq6hu"]
+atlas = ExtResource("1_7p1i2")
+region = Rect2(640, 0, 128, 128)
+
+[sub_resource type="AtlasTexture" id="AtlasTexture_c01r6"]
+atlas = ExtResource("1_7p1i2")
+region = Rect2(768, 0, 128, 128)
+
+[sub_resource type="AtlasTexture" id="AtlasTexture_x3akt"]
+atlas = ExtResource("1_7p1i2")
+region = Rect2(896, 0, 128, 128)
+
+[sub_resource type="AtlasTexture" id="AtlasTexture_is5rt"]
+atlas = ExtResource("2_h360w")
+region = Rect2(0, 0, 128, 128)
+
+[sub_resource type="AtlasTexture" id="AtlasTexture_jq5gw"]
+atlas = ExtResource("2_h360w")
+region = Rect2(128, 0, 128, 128)
+
+[sub_resource type="AtlasTexture" id="AtlasTexture_r603x"]
+atlas = ExtResource("2_h360w")
+region = Rect2(256, 0, 128, 128)
+
+[resource]
+animations = [{
+"frames": [{
+"duration": 1.0,
+"texture": SubResource("AtlasTexture_egmya")
+}, {
+"duration": 1.0,
+"texture": SubResource("AtlasTexture_0mrsi")
+}, {
+"duration": 1.0,
+"texture": SubResource("AtlasTexture_mk0qy")
+}, {
+"duration": 1.0,
+"texture": SubResource("AtlasTexture_jnqno")
+}, {
+"duration": 1.0,
+"texture": SubResource("AtlasTexture_yh4pc")
+}, {
+"duration": 1.0,
+"texture": SubResource("AtlasTexture_kq6hu")
+}, {
+"duration": 1.0,
+"texture": SubResource("AtlasTexture_c01r6")
+}, {
+"duration": 1.0,
+"texture": SubResource("AtlasTexture_x3akt")
+}],
+"loop": true,
+"name": &"default",
```

Comment:
I wonder if we can come up with a convention for spriteframes animation names for props, like we have "idle", "walk", etc for characters. So they are exchangable. I think this works to start.